src/components/patterns/on-the-page.tsx

```diff
@@ -0,0 +1,136 @@
+"use client"
+
+import React, {useState, useEffect} from "react"
+import {NodeStanfordNews, NodeStanfordPage, NodeSulLibrary} from "@/lib/gql/__generated__/drupal.d"
+import {twMerge} from "tailwind-merge"
+import DrupalLink from "./elements/drupal-link"
+import SelectList from "./elements/select-list"
+import {SelectOptionDefinition} from "@mui/base/useSelect"
+
+interface Heading {
+  text: string
+  id: string
+}
+
+const OnThePageLink = ({node}: {node: NodeStanfordPage | NodeStanfordNews | NodeSulLibrary}) => {
```

Comment:
Since this is a client component, make sure to pass as minimal props down to it. In fact, you can even use the `children` prop and render that server side.
```
<OnThePageLink>
<h2>heading</h2>
<a>link1</a>
etc
</OnThePageLink>
```

Reply:
We do need the relLinkHeading and relLinks for the tablet/mobile SelectList so I'll pass in those specific props instead 😃